source/common/router/config_impl.h

```diff
@@ -902,8 +852,8 @@ class RouteEntryImplBase : public RouteEntry,
   const std::string prefix_rewrite_;
   Regex::CompiledMatcherPtr regex_rewrite_;
   Regex::CompiledMatcherPtr regex_rewrite_redirect_;
-  const PathMatchPolicyImpl path_match_policy_;
-  const PathRewritePolicyImpl path_rewrite_policy_;
+  const PathMatcherSharedPtr extension_path_matcher_;
```

Comment:
nit: ditto about removing the "extension_" prefix.

Reply:
There is already a path_matcher_ . I changed over extension_path_rewriter_

Reply:
Where is the other `match_matcher_` member?  I don't see it in this class's members list. How does it differ from this new member? I see we have a new `pathMatcher()` method which returns the `extension_` member. But they should probably match in names right?

Reply:
changed